Make.config

```diff
@@ -60,7 +60,7 @@ MIN_MONO_URL=https://bosstoragemirror.blob.core.windows.net/wrench/mono-2017-04/
 # Minimum Visual Studio version
 MIN_VISUAL_STUDIO_URL=https://bosstoragemirror.blob.core.windows.net/wrench/monodevelop-lion-dogfood-vNext/8f/8f1c13cb983138ee63bd53e09908ea5e737988cd/VisualStudioForMac-Preview-7.0.0.2728.dmg
 MIN_VISUAL_STUDIO_VERSION=7.0.0.2728
-MAX_VISUAL_STUDIO_VERSION=7.2.99
+MAX_VISUAL_STUDIO_VERSION=7.3.99
```

Comment:
Why is this required? I think we should keep the same as 15.3 unless we **require** this